core/src/test/scala/akka/persistence/cassandra/EventsByTagRestartSpec.scala

```diff
@@ -83,5 +84,79 @@ class EventsByTagRestartSpec extends CassandraSpec(EventsByTagRestartSpec.config
       tagProbe.expectNoMessage(waitTime)
       tagProbe.cancel()
     }
+
+    "not write incorrect tag sequence nrs when actor is stopped prematurely" in {
+      val tag = "green"
+      val queryJournal = PersistenceQuery(system).readJournalFor[CassandraReadJournal](CassandraReadJournal.Identifier)
+      val p2 = system.actorOf(TestTaggingActor.props("p2", Set(tag)))
+      val probe1 = TestProbe()
+      probe1.watch(p2)
+      p2.tell("e1", probe1.ref)
+      probe1.expectMsg(Ack)
+      p2.tell("e2", probe1.ref)
+      probe1.expectMsg(Ack)
+      p2.tell("e3", probe1.ref)
+      probe1.expectMsg(Ack)
+      p2.tell("e4", probe1.ref)
+
+      // the PoisonPill will cause the actor to be stopped before the journal write has been completed,
+      // and is therefore similar to if the CircuitBreaker would trigger
+      p2 ! PoisonPill
+      probe1.expectTerminated(p2)
+      val greenTags = queryJournal.eventsByTag(tag, offset = NoOffset)
+      val tagProbe = greenTags.runWith(TestSink.probe[Any](system))
+      tagProbe.request(10)
+      (1 to 3).foreach { n =>
+        val event = s"e$n"
+        system.log.debug("Expecting event {} sequenceNr {}", event, n)
+        tagProbe.expectNextPF { case EventEnvelope(_, "p2", `n`, `event`) => }
+      }
+
+      // without the fix this would see a fourth element (with the wrong tagSeqNr)
+      tagProbe.expectNoMessage(waitTime)
```

Comment:
right, it will typically on be 3, but it could be 4 if the write was faster than the poison
maybe just remove this assert

Reply:
I added a no-element or if there was one it should be the right one. Not super useful but should pass.